definitions/procedures/foreman_tasks/resume.rb

```diff
@@ -8,6 +8,8 @@ class Resume < ForemanMaintain::Procedure
 
     def run
       output << feature(:foreman_tasks).resume_task_using_hammer
+      puts '[ Waiting 30 seconds for resumed tasks to start. ]'
+      sleep 30
```

Comment:
Will it be good to define constant at either procedure level or feature level?

Reply:
Yes, we can do that.